src/test/resources/pages/studentFeedbackSubmitPageGracePeriod.html

```diff
@@ -66,7 +66,7 @@
               </label>
               <div class="col-sm-10">
                 <p class="form-control-static">
-                  ${datetime.now}
+                  ${datetime.now} UTC+0800
```

Comment:
Nope this was as planned. ${datetime.now} only replaces the current date in the format in which it is displayed in the FS Submit Page. The `UTC +0800` will not change depending on when the test will run.